vendor/github.com/longhorn/backing-image-manager/pkg/types/types.go

```diff
@@ -9,18 +12,17 @@ const (
 	DiskPathInContainer              = "/data/"
 	DataSourceDirectoryName          = "/tmp/"
 
-	EnvPodIP = "POD_IP"
-
 	DefaultSectorSize = 512
 
 	DefaultManagerPort              = 8000
-	DefaultDataSourceServerPort     = 8001
+	DefaultDataSourceServerPort     = 8000
+	DefaultSyncServerPort           = 8001
 	DefaultVolumeExportReceiverPort = 8002
 
-	GRPCServiceTimeout     = 3 * time.Minute
-	HTTPTimeout            = 4 * time.Second
-	FileValidationInterval = 5 * time.Second
-	FileSyncTimeout        = 120
+	GRPCServiceTimeout = 3 * time.Minute
```

Comment:
Just curious. How to determine how long the timeout should be?

Reply:
Nice catch.
Typically it depends on if there is a blocking gRPC API and how long it can take.
But here there should be no blocking call since we should use a shorter timeout. Or we need to reuse the client/connection (similar to longhorn-engine/longhorn-instance-manager parts). I will add a TODO here.